added growth sessions guide (#4309)
PostHog/posthog.com: contents/handbook/growth/growth-engineering/growth-sessions.md

They have led to some significant increases in monthly revenue growth.

!> Disclaimer: these are worth doing post product market fit. Before then, just focus on making something people want, rather than optimizing.

## What it is

### Before the meeting
* Before the meeting, everyone creates a list of hypotheses. These can be problems or opportunities. They should all be growth engineering related, and they should be focused on improving our long term monthly growth rate.
* Examples:
    * "More expensive new pricing has hurt signups"
    * "We're not solving bugs quickly and this is hurting word of mouth growth"
    * "We are losing b2c customers to open source"
    * "We should charge for products separately so we can upsell on features rather than volume. This would let us charge sooner than our current 1m events/month limit, and would let us keep prices low for people that don't use the full breadth of the platform"

### When the meeting starts
* When the meeting starts, allocate 10 minutes for people to read through everything and to add any hypotheses if they've not prepared.

> * When the meeting has started, watch some session recordings of users going through key flows of the product. This may generate some further hypotheses. For example, users converting to paying customers, users activating, users inviting their colleagues.

Reviewer: ```suggestion
* After reading, watch session recordings of users going through key flows of the product. Allocate 20 minutes for this. This may generate some further hypotheses. For example, users converting to paying customers, users activating, users inviting their colleagues.
```

* Then work through each hypotheses. Answer each hypothesis by looking at the data and write up any actions (for example, experiments we want to run).

### After the meeting

> * 2 weeks after the meeting, we run a growth sprint planning session to retrospect on what got done and to confirm what we're doing next

Reviewer: ```suggestion
* 2 weeks after the meeting, we run a growth sprint planning session to retrospect on what got done and to confirm what we're doing next.
```

> * 4 weeks later, we re-run the entire meeting. You may end up carrying some hypotheses from session to session (ie if they're blocked or lower priority).

Reviewer: ```suggestion
* 4 weeks later, we re-run the growth session meeting. You may end up carrying some hypotheses from session to session (ie if they're blocked or lower priority).
```

Reviewer: clarifying which type of meeting is re-run

Examples:

    * "New, more expensive pricing has hurt signups"

    * "We're not solving bugs quickly and this is hurting word of mouth growth"

    * "We are losing B2C customers to open source"

    * "We should charge for products separately so we can upsell on features rather than volume. This would let us charge sooner than our current 1m events/month limit, and would let us keep prices low for people that don't use the full breadth of the platform"

### During the meeting
* When the meeting starts, allocate 10 minutes for people to read through everything and to add any additional hypotheses.

* After reading, watch session recordings of users going through key flows of the product. Allocate 20 minutes for this. This may generate some further hypotheses. For example, users converting to paying customers, users activating, users inviting their colleagues.

* During the meeting, you should work through each hypothesis to determine if it is correct and decide how to respond. Answer each hypothesis using available data and write up any actions, such as experiments you want to run, with clearly allocated owners. 

### After the meeting

> * Two weeks after the meeting, run a growth sprint planning session to reflect on what got done and to confirm what comes next.

Author: ```suggestion
```
